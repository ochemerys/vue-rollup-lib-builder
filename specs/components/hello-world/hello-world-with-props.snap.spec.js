import { mount } from '@vue/test-utils';

// @ points to src folder
// no need to add .vue extension
import HelloWorld from '@/lib-components/hello-world.vue'; 

describe('HelloWorld component with props snapshot', () => {
  // create snapshot to match requirements
  it('should mount HelloWorld component with params', () => {
    // create component wrapper and pass props
    const wrapper = mount(HelloWorld, {
      propsData: {
        value: 'Tester'
      }
    });
    // create component snapshot
    expect(wrapper.html()).toMatchSnapshot();
  });
});

