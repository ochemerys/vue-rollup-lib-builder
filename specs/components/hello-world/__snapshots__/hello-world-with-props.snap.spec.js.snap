// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld component with props snapshot should mount HelloWorld component with params 1`] = `<div class="hello" style="color: green;">Hello Tester.</div>`;
